docs/products/opensearch/troubleshooting/troubleshooting-opensearch-dashboards.rst

```diff
@@ -0,0 +1,15 @@
+Troubleshoot opensearch dashboards
+==================================
+
+OpenSearch® Dashboards is unavailable
+---------------------------------------
+
+OpenSearch® Dashboards requires the version of OpenSearch® on your cluster to be the same version. If OpenSearch® Dashboards is unavailable
+
+You may see the following logs in your opensearch-dashboards
```

Comment:
OpenSearch Dashboards requires that the version of OpenSearch on your cluster be the same. If OpenSearch Dashboards is unavailable, you will see the following error message in the opensearch-dashboards logs: